polkadot/runtime/common/src/impls.rs

```diff
@@ -159,30 +171,30 @@ pub mod benchmarks {
 	impl AssetKindFactory<VersionedLocatableAsset> for AssetRateArguments {
 		fn create_asset_kind(seed: u32) -> VersionedLocatableAsset {
 			VersionedLocatableAsset::V3 {
```

Comment:
this should better be latest, V4

Reply:
Yes, I'm going to make LocatableAssetId a versioned_type